index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+
```

Comment:
Video presentation of your project is missing. Kindly recorded a video presenting the features of the project:
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video contains a demo of the project, showing the pages in different screen sizes.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.